eng/pipelines/templates/jobs/perf.yml

```diff
@@ -0,0 +1,121 @@
+parameters:
+  - name: serviceDirectory
+    type: string
+    default: ''
+  - name: services
+    type: string
+    default: ''
+  - name: packageVersions
+    type: string
+    default: '.*'
+  - name: tests
+    type: string
+    default: ''
+  - name: arguments
+    type: string
+    default: '' 
+  - name: iterations
+    type: number
+    default: '5'
+  - name: envVars
+    type: object
+    default: {}
+  - name: additionalArguments
+    type: string
+    default: ''
+
+variables:
+  - name: NugetSecurityAnalysisWarningLevel
+    value: 'none'
+  - name: skipComponentGovernanceDetection
+    value: 'true'
+  - name: DisableDockerDetector
+    value: 'true'
+  - name: Pool
+    value: 'azsdk-pool-mms-ubuntu-2004-perf'
+  - name: OSVmImage
+    value: 'MMSUbuntu20.04'
+
+resources:
+  repositories:
+  - repository: azure-sdk-tools
+    type: github
+    endpoint: Azure
+    name: Azure/azure-sdk-tools
+
+jobs:
+- job: Perf
+  timeoutInMinutes: 360
+  pool:
+    name: $(Pool)
+    vmImage: $(OSVmImage)
+  steps:
+    - checkout: self
+      path: s
+
+    - checkout: azure-sdk-tools
+      path: s/azure-sdk-tools
+
+    - template: /eng/common/pipelines/templates/steps/verify-agent-os.yml
+      parameters:
+        AgentImage: $(OSVmImage)
+
+    - task: JavaToolInstaller@0
+      inputs:
+        versionSpec: '8'
+        jdkArchitectureOption: 'x64'
+        jdkSourceOption: 'PreInstalled'
+      displayName: Install Java 8
```

Comment:
Is this to make a guarantee that Java 8 is installed?

Reply:
It ensures Java 8 is installed, and also makes Java 8 the default when running commands.  In our other pipelines we specify the Java version in the Maven task.

https://github.com/Azure/azure-sdk-for-java/blob/main/eng/pipelines/templates/jobs/ci.yml#L186

But the perf automation shells out to Maven, so the java version needs to be set this way.